tests/bb-modules/Servicehosting/ServiceTest.php

```diff
@@ -784,7 +784,7 @@ public function testtoApiArray()
         $hostingServer = new \Model_ServiceHostingServer();
         $hostingServer->loadBean(new \RedBeanPHP\OODBBean());
         $hostingServer->manager = 'Custom';
-        $hostingHp = new\Model_ServiceHostingHp();
+        $hostingHp = new \Model_ServiceHostingHp();
```

Comment:
Explain the spacing.

Reply:
That's how we create an object from a class? It supposed to have a space between the "new" keyword and the class name. Else, it throws out an error in PHP 8.

Reply:
Thanks for the explanation.